Formula/g/gnome-builder.rb

```diff
@@ -0,0 +1,106 @@
+class GnomeBuilder < Formula
+  desc "Develop software for GNOME"
+  homepage "https://apps.gnome.org/Builder"
+  url "https://download.gnome.org/sources/gnome-builder/46/gnome-builder-46.2.tar.xz"
+  sha256 "0c857b89003b24787f2b1d2aae12d275a074c6684b48803b48c00276d9371963"
+  license "GPL-3.0-or-later"
+
+  head do
+    url "https://gitlab.gnome.org/GNOME/gnome-builder.git", branch: "main"
+    depends_on "gom"
+  end
+
+  depends_on "desktop-file-utils" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "portal" => :build
+  depends_on "vala" => :build
+  depends_on "adwaita-icon-theme"
+  depends_on "cairo"
+  depends_on "cmark"
+  depends_on "editorconfig"
+  depends_on "enchant"
+  depends_on "glib"
+  depends_on "gobject-introspection"
+  depends_on "gtk4"
+  depends_on "gtksourceview5"
+  depends_on "icu4c" # GNOME Builder uses dylibs from icu4c not included in macOS
```

Comment:
`brew audit --new gnome-builder` returns "Dependency 'icu4c' is provided by macOS; please replace 'depends_on' with 'uses_from_macos'.", but, see this comment.